### PR TITLE
Feature/postgres

### PR DESCRIPTION
**Pull request by laggery**
Changes for Postgres migration

### CODE REVIEW

**Comment by fabiangotzen** [comment]
Maybe we could also directly add the docker-compose file with the postgres instance to the project? e.g in a toplevel docker folder.

File: src/db/migrations/1613943227738-pg.ts [hunk 1]
```diff
@@ -0,0 +1,123 @@
+import {MigrationInterface, QueryRunner} from "typeorm";
```
Comment (fabiangotzen) [inline src/db/migrations/1613943227738-pg.ts:1]: formatting:  { MigrationInterface, QueryRunner }

File: src/flight/flight.entity.ts [hunk 2]
```diff
@@ -1,86 +1,82 @@
...
+  @JoinColumn([{ name: "landing_id", referencedColumnName: "id" }])
+  landing: Place;
 
-    @Column("int",{ 
-        nullable:true,
-        name:"km"
-        })
-    km:number | null;
-        
+  @ManyToOne(() => Place, (place) => place.flights2, {
```
Comment (fabiangotzen) [inline src/flight/flight.entity.ts:69]: Maybe we could change the naming of flights2 here?

File: src/user/user.entity.ts [hunk 3]
```diff
@@ -1,177 +1,100 @@
...
+})
+@Index("idx_16606_uniq_8d93d649a0d96fbf", ["emailCanonical"], { unique: true })
+@Index("idx_16606_primary", ["id"], { unique: true })
+@Index("idx_16606_uniq_8d93d64992fc23a8", ["usernameCanonical"], {
+  unique: true,
+})
+@Index("idx_16606_idx_ed85f2a29d842b9b0dfefaba34", ["usernameCanonical"], {
+  unique: true,
+})
+@Entity("user")
```
Comment (fabiangotzen) [inline src/user/user.entity.ts:19]: Does this work without problems? I remember having problems with 'User' Tables in Postgres. But might just be something specific to Spring / JPA :)

Comment (laggery) [reply src/user/user.entity.ts:19]: I also had a problem with this table. User is a reserved table in Postgres. It is however possible to use it but you have to specify the schema used

**Pull request merged**